nipype/utils/draw_gantt_chart.py

```diff
@@ -102,61 +102,14 @@ def log_to_dict(logfile):
     '''
 
     # Init variables
-    #keep track of important vars
-    nodes_list = [] #all the parsed nodes
-    unifinished_nodes = [] #all start nodes that dont have a finish yet
-
     with open(logfile, 'r') as content:
-        #read file separating each line
-        content = content.read()
-        lines = content.split('\n')
-
-        for l in lines:
-            #try to parse each line and transform in a json dict.
-            #if the line has a bad format, just skip
-            node = None
-            try:
-                node = json.loads(l)
-            except ValueError:
-                pass
-
-            if not node:
-                continue
-
-            #if it is a start node, add to unifinished nodes
-            if 'start' in node:
-                node['start'] = parser.parse(node['start'])
```

Comment:
You can remove the `parser` import from this file.

Reply:
done